app/views/images/show.html.haml

```diff
@@ -0,0 +1,27 @@
+%div{:id => "images"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace